packages/mgt-components/src/components/mgt-file-list/mgt-file-list.ts

```diff
@@ -36,6 +36,7 @@ import {
 import '../sub-components/mgt-spinner/mgt-spinner';
 import { OfficeGraphInsightString, ViewType } from '../../graph/types';
 import { styles } from './mgt-file-list-css';
+import { strings } from './strings';
```

Comment:
I am not sure why these changes are showing up. I did make a mistake that I branched from Nic's localization branch vs next/file. But that has been merged..... JFYI